src/app/modules/shared_models/active_section.nim

```diff
@@ -17,11 +17,13 @@ QtObject:
 
   proc membersChanged*(self: ActiveSection) {.signal.}
   proc pendingRequestsToJoinChanged*(self: ActiveSection) {.signal.}
+  proc historyArchiveSupportEnabledChanged*(self: ActiveSection) {.signal.}
 
   proc setActiveSectionData*(self: ActiveSection, item: SectionItem) =
     self.item = item
     self.membersChanged()
     self.pendingRequestsToJoinChanged()
+    self.historyArchiveSupportEnabledChanged()
```

Comment:
@PascalPrecht the only two places where we deal with `ActiveSection` are `modules/main/view.nim` via `activeSection` property for which `activeSectionChanged` signal is emitted and `modules/main/communities/view.nim` via `observedCommunity` property for which `observedItemChanged` signal is emitted, that means that all sub properties will be reevaluated as well. (The same states for two signals above `membersChanged ` and `pendingRequestsToJoinChanged `)